newsroom/cards/module.py

```diff
@@ -26,11 +27,14 @@
 async def get_settings_data():
     app = get_current_wsgi_app()
 
+    cursor = await NavigationsService().search(lookup={"is_enabled": True})
```

Comment:
In situations like this, where we have multiple `await` for separate DB requests, we can probably start using `asyncio.gather` to run both requests concurrently and await it's results.

In this case, it would be:
```python
from asyncio import gather

async def get_settings_data():
    ...
    cursors = await gather(
        NavigationsService().search({"is_enabled": True}),
        CardsResourceService().search({}),
    )
    [navigations, cards] = await gather(
        cursors[0].to_list_raw(),
        cursors[1].to_list_raw(),
    )

    return {
        ...,
        "cards": cards,
        "navigations": navigations,
    }
```

Reply:
good idea. I've made the changes.